pagerduty.py

```diff
@@ -139,10 +140,10 @@ def _queued_files(self):
         # We need to sort the files by the timestamp.
         # This function extracts the timestamp out of the file name
         def file_timestamp(file_name):
-            return int(re.search('pd_(\d+)_', file_name).group(1))
+            return int(re.search('pd_(\d+)', file_name).group(1))
 
-        sorted_file_names = sorted(pd_file_names, key=file_timestamp)
-        return pd_file_names
+        sorted_file_names = sorted(pd_file_names, key=file_timestamp, reverse=True)
```

Comment:
I think due to the queued files and because I wasn't using the cronjob to flush the queue, I ended up sending the files wrong in the wrong order by name but they looked right when viewing the trigger/resolve API calls. I'll fix the ordering and make the few other changes.